README.md

```diff
@@ -12,15 +12,20 @@ var log = new LoggerConfiguration()
     .WriteTo.DatadogLogs("<API_KEY>")
     .CreateLogger();
 ```
+## Configuration
 
-By default the logs are forwarded to Datadog via **HTTPS** on port 443 to the US site.
-You can change the site to EU by using the `url` property and set it to `https://http-intake.logs.datadoghq.eu`.
+By default the logs are forwarded to Datadog via **HTTPS** on port 443 to the US site: `https://http-intake.logs.datadoghq.com)`.  
+You can change the site to EU by using the `Url` property on the `DatadogConfiguration` object and set it to `https://http-intake.logs.datadoghq.eu`. This can be done by passing in the url in the constructor, or by setting the property directly.
```

Comment:
```suggestion
You can change the site to EU by using the `Url` property on the `DatadogConfiguration` object and set it to `https://http-intake.logs.datadoghq.eu`. This can be done by passing in the URL in the constructor, or by setting the property directly.
```